tests/test_api.py

```diff
@@ -2527,7 +2539,8 @@ def test_rebuild_hashtable_chunk_reuse_unicode(tmp_path, caplog):
 
 
 def test_rebuild_hashtable_chunk_reuse_multi_dim(tmp_path, caplog):
-    """Test that the correct chunks are used after rebuilding the tables for a multi-dimensional array."""
+    """Test that the correct chunks are used after rebuilding the tables for a
+    multi-dimensional array."""
```

Comment:
```suggestion
    multi-dimensional array.
    """
```

I think multiline docstrings should have the terminating `"""` on a separate line.